randovania/games/prime2/json_data/Torvus Bog.txt

```diff
@@ -2928,7 +2941,9 @@ Extra - in_dark_aether: False
   > Room Center
       Any of the following:
           Gravity Boost
-          Morph Ball Bomb and Morph Ball and Screw Attack and Space Jump Boots and Air Underwater (Expert) and Bomb Space Jump (Expert)
+          All of the following:
+              # https://www.youtube.com/watch?v=LGgiOIFtmY4
+              Morph Ball Bomb and Morph Ball and Screw Attack and Space Jump Boots and Air Underwater (Expert) and Bomb Space Jump (Expert)
```

Comment:
Just for simplification purposes, change this to use the Use SA (SJ) template instead of listing Morph, SA, SJ.

Reply:
Done.